assets/node-exporter/daemonset.yaml

```diff
@@ -39,6 +41,10 @@ spec:
           mountPropagation: HostToContainer
           name: root
           readOnly: true
+        - mountPath: /etc/node_exporter/textfile
+          name: node-exporter-textfile
+          readOnly: false
```

Comment:
I don't see any reason why node-exporter should have write access to this dir.

Reply:
good point